src/devices/Media/SoundDevice/Devices/UnixSoundDevice.cs

```diff
@@ -147,6 +159,24 @@ public override void Record(uint recordTimeSeconds, Stream outputStream)
             IntPtr @params = new IntPtr();
             int dir = 0;
 
+            var header = CreateHeader(recordTimeSeconds);
+
+            WriteWavHeader(outputStream, header);
+            try
+            {
+                OpenRecordingPcm();
+                PcmInitialize(_recordingPcm, header, ref @params, ref dir);
+                ReadStream(outputStream, header, ref @params, ref dir);
+                CloseRecordingPcm();
+            }
+            finally
+            {
+                Dispose();
```

Comment:
why do you dispose from Record method?

Reply:
It was the original logic and didn't changed it. It's not really necessary. So will remove.